src/include/miopen/solver.hpp

```diff
@@ -82,6 +82,15 @@ struct SolverBase
     /// overriden to keep the name to avoid DB corruption.
     virtual const std::string& SolverDbId() const = 0;
 
+    // In some instances ( particularly fusions) the fused solver might like to
+    // fallback to the non-fused variant for performance parameters, this information
+    // is returned via AltSolverDbId
```

Comment:
```suggestion
    /// In some instances ( particularly fusions) the fused solver might like to
    /// fallback to the non-fused variant for performance parameters, this information
    /// is returned via AltSolverDbId
```

Reply:
I am not sure what is the advantage of the suggested change.